recipe/build.sh

```diff
@@ -6,4 +6,5 @@ dst=../test_files
 rm -fr $dst
 mkdir -pv $dst
 mv -v $PKG_NAME $dst/
+ln -s $dst/$PKG_NAME
```

Comment:
Fixes a problem observed when running `make package`, related to the recent introduction of the `--version` switch and its use of `src/uwtools/resources/info.json`.